### ISSUE TITLE
Copy Folder from vscode SSH environment to local environment

### ISSUE THREAD

**Issue by rmuhawieh** [microsoft/vscode#234373]
Does this issue occur when all extensions are disabled?: Yes/No

I've been searching ways to go about this with no luck. I have a folder saved in the vscode remote server (via ssh) and I'd like to copy the entire folder and have it be in the local vscode environment as well. I've seen scp and downloading extensions suggested but surely there is a simple way through vscode where I'd be able to simple copy from remote vscode environment into the local vscode environment. Thanks

**Comment by technic960183** [comment]
Right-click on the folder and select _Download_; this might solve your issue.
I've been using this method to download a batch of files from a remote server to my local for a while.

**Comment by joshspicer** [comment]
Thanks for the issue.  You are correct that scp or using the 'download' context menu button would be the best solutions today. 

/duplicate https://github.com/microsoft/vscode-remote-release/issues/2008
